src/Features/Core/Portable/ExternalAccess/UnitTesting/API/INewUnitTestingIncrementalAnalyzerImplementation.cs

```diff
@@ -23,8 +23,22 @@ internal interface INewUnitTestingIncrementalAnalyzerImplementation
         Task AnalyzeSyntaxAsync(Document document, UnitTestingInvocationReasons reasons, CancellationToken cancellationToken);
 #endif
 
-        Task AnalyzeDocumentAsync(Document document, SyntaxNode bodyOpt, UnitTestingInvocationReasons reasons, CancellationToken cancellationToken);
-        Task AnalyzeProjectAsync(Project project, bool semanticsChanged, UnitTestingInvocationReasons reasons, CancellationToken cancellationToken);
+        Task AnalyzeDocumentAsync(
+            Document document,
+#if false // Not used in unit testing crawling
+            SyntaxNode bodyOpt,
+#endif
+            UnitTestingInvocationReasons reasons,
+            CancellationToken cancellationToken);
+
+        Task AnalyzeProjectAsync(
+            Project project,
+#if false // Not used in unit testing crawling
+            bool semanticsChanged,
```

Comment:
Curious why the bool flag was needed when same info could be conveyed via reasons below.

Reply:
great question. i have no answer for that :D